pkgs/applications/audio/tenacity/default.nix

```diff
@@ -0,0 +1,156 @@
+{ stdenv
+, lib
+, fetchgit
+, cmake
+, wxGTK
+, pkg-config
+, python3
+, gettext
+, glib
+, file
+, lame
+, libvorbis
+, libmad
+, libjack2
+, lv2
+, lilv
+, makeWrapper
+, serd
+, sord
+, sqlite
+, sratom
+, suil
+, alsaLib
```

Comment:
`alsaLib` was renamed `alsa-lib` in #126440.

Reply:
Thanks, fixed.